src/models/pickup.js

```diff
@@ -0,0 +1,16 @@
+import InternalUtil from '../utils/internal_util';
+import EasyPostObject from './easypost_object';
+
+export default class Pickup extends EasyPostObject {
+  /**
+   * Get the lowest rate of a pickup.
+   * @param {string} carriers
+   * @param {string} services
+   * @returns {Object}
+   */
+  lowestRate(carriers, services) {
```

Comment:
nit: I would add an empty array default parameter to both carriers and services. This applies to other instance `lowestRate` functions

Reply:
Nice catch

Reply:
why did this get reverted?

Reply:
I rolled it back for all of them. Build failed when these got changed